base/system/logonui/NT5design.c

```diff
@@ -30,7 +30,7 @@ NT5_DrawLogoffCaptionText(LPWSTR lpText,
     LogFont.lfCharSet = DEFAULT_CHARSET;
     LogFont.lfHeight = 22;
     LogFont.lfWeight = 109; // From WinXP disassembly
-    wcscpy_s(LogFont.lfFaceName, LF_FACESIZE, L"Arial");
+    wcscpy(LogFont.lfFaceName, L"Arial");
```

Comment:
Actually I just realized you could use
`StringCchCopyW(LogFont.lfFaceName, _countof(LogFont.lfFaceName), L"Arial");`
(and including the necessary headers)